openedx/core/djangoapps/video_pipeline/migrations/0006_remove_course_percentage_field.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 2.2.14 on 2020-07-10 13:04
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('video_pipeline', '0005_add_vem_course_percentage'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
This is going to cause errors when it gets deployed...due to our deployment model, we will still have code running on some servers that reference this field.  So we need to split the removal into two PRs:  one that removes the code that references the field, and one that removes the field itself.  For more info see https://openedx.atlassian.net/wiki/spaces/AC/pages/23003228/Everything+About+Database+Migrations#EverythingAboutDatabaseMigrations-Deploymentandbackward-compatiblemigrations

Reply:
Yes, that totally slipped my mind. Thanks.